lib/DDG/Spice/SkiResorts.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::SkiResorts;
+
+# ABSTRACT: Displays information on ski resorts
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use JSON;
+
+# Configuration
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint
+spice to => 'http://www.piste.io/info/$1.json';
+
+# Load in resort list for triggers
+my $data = share('resorts.json')->slurp;
+my @resorts = @{decode_json($data)};
+@resorts = map { $_ =~s/-/ /g; $_; } @resorts; # Replace dashes with spaces
+triggers any => @resorts;
```

Comment:
OK, fair point. I will make the change, adding more specific triggers. I do think there is a balance to be struck here though, we do want balance false positives with discovery. For example I would never type something as verbose as `ski conditions at Verbier` into a search engine, I would just have `ski Verbier`, knowing that the results would contain what I'd like